src/transformers/models/llama/modeling_llama.py

```diff
@@ -115,15 +114,15 @@ def __init__(self, dim, max_position_embeddings=2048, base=10000, device=None, s
     @property
     def sin_cached(self):
         logger.warning_once(
-            "The sin_cached attribute will be removed in 4.39. Bear in mind that its contents changed in v4.38. Use "
+            "The sin_cached attribute will be removed in 4.42. Bear in mind that its contents changed in v4.40. Use "
```

Comment:
I see this is for copied from but let' s just remove it and not use copied from for both of cos and sin cached